spec/ngsild_swagger.json

```diff
@@ -0,0 +1,1077 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "description": "This swagger file describes the NGSI-LD API defined by the ETSI ISG CIM group. This Cross-domain Context Information Management API allows to provide, consume and subscribe to context information in multiple scenarios and involving multiple stakeholders",
```

Comment:
can we have this in multiple lines?

Reply:
I will check this after successfully parse the files with a validator